docs/package.json

```diff
@@ -3,16 +3,70 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "build": "npm run build-html && npm run build-css && npm run copy",
-    "build-html": "mkdir ../out; node index.js > ../out/index.html",
-    "build-css": "mkdir -p ../out/styles; cat ./styles/third_party/bulma.css ./styles/site.css | cleancss -o ../out/styles/bundle.css",
-    "copy": "cp 404.html CNAME robots.txt ../out && cp -r images ../out && cp -r libraries ../out"
+    "build": "wireit",
+    "build-html": "wireit",
+    "build-css": "wireit",
+    "copy": "wireit"
   },
   "author": "Rob Dodson",
   "license": "MIT",
   "devDependencies": {
     "clean-css-cli": "5.6.1",
     "handlebars": "4.7.7",
     "marked": "0.8.2"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build-html",
+        "build-css",
+        "copy"
+      ]
+    },
+    "build-css": {
+      "files": [
+        "styles/third_party/bulma.css",
+        "styles/site.css"
+      ],
+      "commentedOut-output": [
```

Comment:
Why are the outputs commented out?

Reply:
They're outside of this package directory.

Caching won't happen for the slow parts of these scripts anyways because they depend on the ../:build script, which reads github stars and so can't be cached.

Reply:
In a future change I might have these scripts emit an out directory in docs and have the top level build script copy that up to the toplevel out dir, but didn't seem worth the hassle right now